### PR TITLE
Cancel withdrawal

### PR DESCRIPTION
**Pull request by uk4z**
# What
Adds the possibility to cancel withdrawals in our frontend. 

# How
Major changes: 
- new file: CancelWarning.svelte
- new icon: RevertIcon.svelte
- changed the storing logic in Withdrawals.svelte to make the cancelled withdrawals after the active ones
- poorly named function fix 
- refactored some utils functions to the lib

### CODE REVIEW

**Review by uk4z — COMMENTED** [review]
I did not spot any logic problems. I shall implement tests.

**Review by uk4z — COMMENTED** [review]
lgtm, the tests passed.

**Review by uk4z — COMMENTED** [review]
Changed the layout a bit.

**Review by uk4z — COMMENTED** [review]
lgtm

**Review by leo-ship-it — APPROVED** [review]

File: src/lib/icons/RevertIcon.svelte [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	version="1.1"
+	xmlns="http://www.w3.org/2000/svg"
+	height="24"
+	width="24"
+	><g id="SVGRepo_bgCarrier" stroke-width="0"></g><g
+		id="SVGRepo_tracerCarrier"
+		stroke-linecap="round"
+		stroke-linejoin="round"
+	></g><g id="SVGRepo_iconCarrier">
+		<title>cancel2</title>
```
[the remark this answers is not in the document]
Comment (uk4z) [reply src/lib/icons/RevertIcon.svelte:13]: got rid of it

**Pull request merged**